ngmix/bootstrap.py

```diff
@@ -279,93 +274,6 @@ def _get_round_pars(self, pars_in):
 
         return pars, pars_lin
 
-
-    def _find_cen(self, ntry=10, Tguess=4.0):
```

Comment:
What happened to this method? Do we not need it anymore?

Reply:
no, it was never used in production